src/app/components/client/ExposureCard.tsx

```diff
@@ -70,23 +50,254 @@ type BreachExposureCategoryProps = {
   emails?: string[];
 };
 
-type ScannedExposureCategoryProps = {
-  exposureCategoryLabel: string;
-  num: number;
-  icon: ReactElement;
+export const ExposureCard = ({ exposureData, ...props }: ExposureCardProps) => {
+  return isScanResult(exposureData) ? (
+    <ScanResultCard {...props} scanResult={exposureData} />
+  ) : (
+    <SubscriberBreachCard {...props} subscriberBreach={exposureData} />
+  );
+};
+
+export type ScanResultCardProps = {
+  exposureImg?: StaticImageData;
+  scanResult: ScanResult;
+  locale: string;
+  isPremiumBrokerRemovalEnabled: boolean;
+};
+const ScanResultCard = (props: ScanResultCardProps) => {
```

Comment:
Most of this "new" code is a verbatim copy from the old `<ExposureCard>` component, but only the code branches related to scan results.